servers/rest-services/src/main/java/org/projectnessie/services/rest/RestContentResource.java

```diff
@@ -50,12 +52,14 @@ private ContentService resource() {
   }
 
   @Override
+  @JsonView(Views.V1.class)
```

Comment:
We should probably put them on V1 _interfaces_ too. IIRC, Jersey and Quarkus look for view annotations in different places :shrug:

Reply:
Works w/ Quarkus this way

Reply:
Right, does Jersey see these new `Views.V1` annotations?

Reply:
Tests will not discover that since model classes are backward-compatible for the methods in this PR :thinking: